src/localkiwixserver.cpp

```diff
@@ -43,6 +43,8 @@ LocalKiwixServer::LocalKiwixServer(QWidget *parent) :
     for (const auto &interfacePair : interfacesMap) {
         QString ip = QString::fromStdString(interfacePair.second.addr);
         interfaces.push_back(ip);
+        QString ipv6 = QString::fromStdString(interfacePair.second.addr6);
+        if (!ipv6.isEmpty()) interfaces.push_back(ipv6);
     }
     std::sort(interfaces.begin(), interfaces.end());
     interfaces.push_front(QString(gt("all")));
```

Comment:
Shouldn't we provide "All IPv4" and "All IPv6" options too?

Reply:
Sure, will add it besides All

Reply:
Should I rename to:
- Dual Stack Mode
- IPv4 Mode
- IPv6 mode